src/screens/CreatePost/index.js

```diff
@@ -573,17 +569,13 @@ const CreatePost = () => {
   };
 
   const handleTopicChat = async () => {
-    const defaultImage = DEFAULT_TOPIC_PIC_PATH;
     const allTopics = listTopic;
 
     allTopics.forEach(async (topic) => {
       const channel = await client.client.channel('topics', `topic_${topic}`, {
         name: `#${topic}`,
         members: [user.myProfile.user_id],
-        channel_type: 3,
-        channel_image: defaultImage,
-        channelImage: defaultImage,
-        image: defaultImage
+        channel_type: 3
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The refactoring of the `handleTopicChat` function is well done. The removal of unnecessary properties related to the default topic image makes the code cleaner and more focused on its main functionality. However, please ensure that this change does not affect other parts of the system where these properties might be used.
```diff
- const defaultImage = DEFAULT_TOPIC_PIC_PATH;
- const channel = await client.client.channel('topics', `topic_${topic}`, {
-   name: `#${topic}`,
-   members: [user.myProfile.user_id],
-   channel_type: 3,
-   channel_image: defaultImage,
-   channelImage: defaultImage,
-   image: defaultImage
- });
+ const allTopics = listTopic;
+ allTopics.forEach(async (topic) => {
+   const channel = await client.client.channel('topics', `topic_${topic}`, {
+     name: `#${topic}`,
+     members: [user.myProfile.user_id],
+     channel_type: 3
+   });
+   await channel.create();
+   await channel.addMembers([user.myProfile.user_id]);
+ });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->